examples/stable_diffusion_xl/GETTING_STARTED.md

```diff
@@ -28,13 +28,15 @@ cd tools/model_conversion
 
 # convert sdxl-base-1.0 model
 python convert_weight.py \
+  --task pt_to_ms \
```

Comment:
Please synchronize the modifications to file `weight_convertion.md`
https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/weight_convertion.md